extensions/amp-ad/0.1/amp-ad-ui.js

```diff
@@ -63,8 +63,8 @@ export class AmpAdUIHandler {
     /** {number} */
     this.state = AdDisplayState.NOT_LAID_OUT;;
 
-    /** {?Element} */
-    this.pageProvidedFallback_ = baseInstance.getFallback();
+    /** {!boolean} */
+    this.hasPageProvidedFallback_ = false;
```

Comment:
You're right! And it will cause a bug that `this.hasPageProvidedFallback_` is always `false` with experimental tag set to false. 
I fixed it